frontend_server/test/server_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:async';
+
+import 'package:args/src/arg_results.dart';
+import 'package:frontend_server/server.dart';
+import 'package:test/test.dart';
+
+class _MockedCompiler implements CompilerInterface {
+  Future<Null> Function(String, ArgResults) compileCallback;
+  Future<Null> Function() recompileDeltaCallback;
+  void Function() acceptLastDeltaCallback;
+  void Function() rejectLastDeltaCallback;
+  void Function(Uri) invalidateCallback;
+
+  _MockedCompiler({this.compileCallback, this.recompileDeltaCallback,
+    this.acceptLastDeltaCallback, this.rejectLastDeltaCallback,
+    this.invalidateCallback});
+
+  @override
+  Future<Null> compile(String filename, ArgResults options) =>
```

Comment:
Not sure how to split this method definition up. The rest I can split like this:
```
  void invalidate(Uri uri) => invalidateCallback != null
      ? invalidateCallback(uri)
      : null;
``` 
Hope it's okay.
Also dartfmt does line breaks after `=>`...